app/controllers/ConfirmTransferController.scala

```diff
@@ -55,11 +55,13 @@ class ConfirmTransferController @Inject()(val controllerComponents: SecurityComp
         val confirmTransferStatus = consignmentStatus.flatMap(_.confirmTransfer)
         confirmTransferStatus match {
           case Some("Completed") => Future(Ok(views.html.standard.confirmTransferAlreadyConfirmed(consignmentId, request.token.name)).uncache())
-          case _ =>
+          case None =>
             getConsignmentSummary(request, consignmentId)
               .map { consignmentSummary =>
                 httpStatus(views.html.standard.confirmTransfer(consignmentId, consignmentSummary, finalTransferForm, request.token.name)).uncache()
               }
+          case _ =>
+            throw new IllegalStateException(s"Unexpected Confirm Transfer status: $confirmTransferStatus")
```

Comment:
As a general comment, it might be worth adding the consignment ID into the error message for each of these so we can see which consignment has failed. What do you think?

Reply:
Sure, sounds like a good idea!